### PR TITLE
feat(alert): add the ability to close the alert

### PR DESCRIPTION
**Pull request by poteirard**
- Add brand style - https://www.sketch.com/s/f792aa60-01d9-472d-b53c-c74dd501ef9c?search=Molecules%20%F0%9F%A6%A0%2FBanners%2FDesktop%
- Add ability to close banner - https://www.sketch.com/s/f792aa60-01d9-472d-b53c-c74dd501ef9c/p/banners
https://jira.zopa/browse/ZDS-22

### CODE REVIEW

File: src/components/atoms/Alert/Alert.tsx [hunk 1]
```diff
@@ -81,13 +97,29 @@ const IconWrapper = styled.div<{ severity: Severity }>`
...
   return (
     <Wrapper severity={severity} inline={inline} {...rest}>
       <IconWrapper severity={severity}>
         <Icon />
+        {onRequestClose && (
+          <CrossIcon
+            onClick={onRequestClose}
+            color={text}
+            variant={faTimes}
+            data-testid="ZA.alert-cross-icon"
```
Comment (thegrinder) [inline src/components/atoms/Alert/Alert.tsx:119]: we're sure we want to hardcode it?

Comment (poteirard) [reply src/components/atoms/Alert/Alert.tsx:119]: for now yes, just as the modal one. If we see we need more than one alert in a page we can tweak it later

**Comment by thegrinder** [comment]
you might need a bit more padding on the right if close icon present

image

**Pull request merged**